### PR TITLE
update urls in zh docs default-storage-class-prereqs.md

### PR DESCRIPTION
**Pull request by zhanw15**
url has been changed.

### CODE REVIEW

File: content/zh/includes/default-storage-class-prereqs.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-您需要有一个带有默认[StorageClass](/docs/concepts/storage/storage-classes/)的动态持续卷供应程序，或者自己[静态的提供持久卷](/docs/user-guide/persistent-volumes/#provisioning)来满足这里使用的[持久卷请求](/docs/user-guide/persistent-volumes/#persistentvolumeclaims)。
+您需要有一个带有默认[StorageClass](/content/zh/docs/concepts/storage/storage-classes.md)的动态持续卷供应程序，或者自己[静态的提供持久卷](/content/zh/docs/reference/glossary/persistent-volume.md)来满足这里使用的[持久卷请求](/content/zh/docs/reference/glossary/persistent-volume-claim.md)。
```
Comment (gavinfish) [inline content/zh/includes/default-storage-class-prereqs.md]: 1. Most zh translation docs still keep the links to en version. A general fix for this is under discussion.
1. No need for `.md` in a link.

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_control-plane.md [hunk 2]
```diff
@@ -71,23 +71,3 @@ kubeadm init phase control-plane [flags]
   </tbody>
 </table>
 
-
-
-<!--
```
Comment (gavinfish) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_control-plane.md]: IMO, one PR better has one purpose. The `see also` issue seems to have its own tracking issue.

**Comment by tengqm** [comment]
@zhanw15 Still working on this?

**Comment by zhanw15** [comment]
@tengqm Yes, I'm focusing on this

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_control-plane.md (hunk 2 above)
Comment (tanjunchen) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_control-plane.md:76]: /lgtm
for this change 
I remember that  this issue (removing SEE ALSO ) has been closed

Comment (tanjunchen) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_control-plane.md:76]: see https://github.com/kubernetes/website/issues/18411

**Comment by tengqm** [comment]
/close
This PR has been dangling for too long. Please feel free to reopen it when time permits.

**Pull request closed without merging**